File: c/numeros_pares.c
#include <stdio.h>

int main() {

    int N, qtdpares;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    int vet[N];

    for (int i = 0; i < N; i++) {
        printf("Digite um numero: ");
        scanf("%d", &vet[i]);
    }

    printf("\nNUMEROS PARES:\n");

	qtdpares = 0;
    for (int i = 0; i < N; i++) {
        if (vet[i] % 2 == 0) {
            printf("%d  ", vet[i]);
            qtdpares++;
        }
    }

    printf("\n\nQUANTIDADE DE PARES = %d\n", qtdpares);

    return 0;
}

File: java/numeros_pares.java
import java.util.Scanner;

public class numeros_pares {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		int N, qtdPares;

		System.out.print("Quantos numeros voce vai digitar? ");
		N = sc.nextInt();

		int[] vet = new int[N];

		for (int i = 0; i < N; i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextInt();
		}

		System.out.println("\nNUMEROS PARES:");

		qtdPares = 0;
		for (int i = 0; i < N; i++) {
			if (vet[i] % 2 == 0) {
				System.out.printf("%d  ", vet[i]);
				qtdPares++;
			}
		}

		System.out.printf("\n\nQUANTIDADE DE PARES = %d\n", qtdPares);

		sc.close();
	}
}